src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/StopPoints/Editor/index.tsx

```diff
@@ -1,48 +1,68 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
 import { Tabs, Tab, TabList, TabPanels, TabPanel } from '@entur/tab';
 import { DestinationDisplay, StopPoint } from 'model';
-import { isBlank } from 'helpers/forms';
+import { isBlank, objectValuesAreEmpty } from 'helpers/forms';
 import BookingArrangementEditor from '../../../../BookingArrangementEditor';
 import Header from './Header';
 import './styles.scss';
-import searchForQuay from './searchForQuay';
+import searchForQuay, { QuaySearch } from './searchForQuay';
 import debounce from './debounce';
 import { DEFAULT_SELECT_VALUE } from './constants';
 import Form from './Form';
 import validateForm from './validateForm';
+import FlexibleStopPlace from 'model/FlexibleStopPlace';
 
-class StopPointEditor extends Component {
-  state = {
-    stopPlaceSelection: DEFAULT_SELECT_VALUE,
-    errors: { quayRef: [], flexibleStopPlaceRefAndQuayRef: [], frontText: [] },
-    quaySearch: {}
+export type StopPlaceSelectionType = string | null;
+
+type Props = {
+  isFirst: boolean;
+  stopPoint: StopPoint;
+  onChange: (any: any) => void;
```

Comment:
Är det ikke alltid en `StopPoint` som sänds in?

Reply:
Fixed: 1e203b496e84c57f7ff342283dc34716c7f0f9c8